Makefile

```diff
@@ -19,5 +25,21 @@ prepare:
 	nuget restore
 	(cd external/Java.Interop && nuget restore)
 
+
+run-all-tests: run-tests 
+
 clean:
 	$(MSBUILD) /t:Clean
+
+
+# $(call RUN_TEST,filename,log-lref?)
+define RUN_TEST
+	MONO_TRACE_LISTENER=Console.Out \
+	$(RUNTIME) --runtime=v4.0.0 \
+		$(NUNIT_CONSOLE) $(NUNIT_EXTRA) $(1) \
+		$(if $(RUN),-run:$(RUN)) \
+		-output=bin/Test$(CONFIGURATION)/TestOutput-$(basename $(notdir $(1))).txt ;
+endef
+
+run-tests: $(TESTS)
```

Comment:
This should be `run-nunit-tests`, to differentiate from the `run-apk-tests` in PR #32.